src/com/facebook/buck/swift/SwiftDescriptions.java

```diff
@@ -57,7 +58,10 @@ private SwiftDescriptions() {
       BuildTarget buildTarget) {
 
     output.srcs = filterSwiftSources(sourcePathResolver, args.srcs);
-    output.compilerFlags = args.compilerFlags;
+    if (args.getClass().equals(AppleLibraryDescription.Arg.class))
+      output.compilerFlags = ((AppleLibraryDescription.Arg) args).swiftCompilerFlags;
+    else
+      output.compilerFlags = args.compilerFlags;
```

Comment:
did you run `ant lint`? coz I don't think it allow you to have a condition block without brackets.